scenarios.md

```diff
@@ -262,6 +261,20 @@ A user may want to trust an artifact only if it is signed by a specific key. May
 
 1. Users must be allowed to configure specific trusted keys for specific artifacts or collections of artifacts.
 
+### Scenario #11: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. The signature on the vulnerable artifact is revoked.
+1. Consumers that pulled the vulnerable artifact that verify the signature will receive an error indicating the signer no longer trusts the artifact.
+
+**Implications of this requirement:**
+
+1. Users that wish to still run artifacts that have their signatures revoked will need to disable verification on the vulnerable artifact or resign the artifacts with a key they control and trust.
+1. Attackers attempting to replay old signatures to vulnerable artifacts should be blocked by the verification.
```

Comment:
I implication is not clear to me, I think the revocation status is associated with the artifact and not a signature. We discussed it [here](https://hackmd.io/2rc07WwHTdmPjpgoMbvYfA?view#Artifact-level-revocation)(second point).

Reply:
I don't think the hackmd is part of our scenarios or design yet, that's still a WIP. I'd recommend revoking the signature rather than the full artifact since you may have multiple signatures that mean different things. Revoking a signature that indicates the artifact was approved by sub-team-X for deployment in their cluster shouldn't also revoke the signature that says dev-team-y made this.

Reply:
I agree, it makes sense to associate revocations to signature as the scenario you mentioned and also being a potential attack vector in https://github.com/notaryproject/notaryproject/pull/49#issuecomment-870121767.